docs/projects/lottie/supported.rst

```diff
@@ -462,6 +462,30 @@ Blur Layer
 |     Type     |  |tick| |      |tick|     |
 +--------------+---------+-----------------+
 
+Shade Layer
+-----------
+
++--------------+---------------------+-----------------+
+|   Property   |        Value        | Type(Animation) |
++==============+=====================+=================+
+|    Z_depth   |        |tick|       |     |cross|     |
++--------------+---------------------+-----------------+
+|    Amount    |        |tick|       |      |tick|     |
++--------------+---------------------+-----------------+
+| Blend Method |       |cross|       |     |minus|     |
```

Comment:
Does shade layer not support blend methods at all? I don't remember this actually.
Can someone test the lottie exporter to check this ^__^